pkg/api/networkservice/mechanisms/kernel/constants.go

```diff
@@ -33,6 +35,9 @@ const (
 	// PCIAddressKey - device PCI address property key
 	PCIAddressKey = common.PCIAddressKey
 
+	// TokenIDKey is a token ID property key
+	TokenIDKey = common.TokenIDKey
```

Comment:
Is this the Token we are getting from DevicePlugin in an ENV variable?  If so, could we call it DeviceToken or DeviceTokenID?

Reply:
Yes, sure. Fixed.